br/tests/lightning_extend_routes/config.toml

```diff
@@ -0,0 +1,23 @@
+
+[mydumper]
+source-id = "mysql-01"
+
+# the complicated routing rules should be tested in tidb-tools repo already
+# here we're just verifying the basic things do work.
+[[routes]]
+schema-pattern = "routes_a*"
+table-pattern = "t*"
+target-schema = "routes_b"
+target-table = "u"
+
+    [routes.extract-table]
```

Comment:
It seems it's unnecessary to indent these configurations.

Reply:
Just want to make it clear that these configurations belongs to last part.. I can revise this